scripts/gen_vimdoc.lua

```diff
@@ -372,10 +379,65 @@ local function should_render_param(p)
   return not p.access and not contains(p.name, { '_', 'self' })
 end
 
+--- @param ty string
+--- @param classes? table<string,nvim.luacats.parser.class>
```

Comment:
why do these types deviate from the convention of using `vim.*` prefixed names?

Reply:
Because these files do not live in `$VIMRUNTIME`.

Reply:
Got it. Is this naming convention documented?

Reply:
Nope not yet.